Testscripts/Linux/LSVMBUS.sh

```diff
@@ -146,19 +147,31 @@ else
     expected_network_counter=$VCPU
 fi
 
-# the cpu count that attached to the SCSI driver is MIN(the number of vCPUs/4, 64).
-if [ "$VCPU" -gt 64 ];then
-    expected_scsi_counter=64
-else
-    expected_scsi_counter=$(bc <<<"scale=2;$VCPU/4")
-    if [ "$expected_scsi_counter" != "${1#*.00}" ]; then
-        # In this case we have a float number that needs to be rounded up.
-        # For example with 6 cores, scsi controller is spread on 2 cores.
-        expected_scsi_counter=$(bc <<<"("$expected_scsi_counter"+0.5)/1")
-    fi
-    # normalizing the number to integer
-    expected_scsi_counter=${expected_scsi_counter%.*}
-fi
+case $VM_Size in
+    # If Lv2, the cpu count that attached to the SCSI driver is MIN(the number of vCPUs, 64).
+    *Standard_L*s_v2*)
```

Comment:
this case support both on Azure and HyperV, sometimes our size mapping are not exact correct, then that would be a problem when test on hyperv

Reply:
Does it mean we should run conditionally for Azure and Hyper-v?

Reply:
@yuxisun1217 You only run case with override size on azure platform, right?

Reply:
Hi @LiliDeng,
Thank you for pointing out that! I didn't consider about Hyper-V actually. I'll trying to add a platform condition, if Hyper-V, set VM_size=None. Except that, do you think it can be on Hyper-V? I'm still not quite familiar with the .ps1 script in Hyper-V...I'm looking for a Hyper-V testbed to test it.

Reply:
@LiliDeng ,
For the override size, yes, I only test it on Azure.

Reply:
@yuxisun1217 if default size work well against Hyper-V, then it is fine, let me run it and then see if it break default size on hyperv or not.

Reply:
Hi @LiliDeng ,
I just run test in Hyper-V with RHEL-8.2 and here is the result:

[LISAv2 Test Results Summary]
Test Run On           : 05/29/2020 09:29:15
VHD Under Test        : C:\vmtest\Virtual Hard Disks\RHEL-8.2.0-updates-20200423.0-x86_64-GEN1-A.vhdx
Initial Kernel Version: 4.18.0-193.el8.x86_64
Final Kernel Version  : 4.18.0-193.el8.x86_64
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:2

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 CORE                 LSVMBUS                                                                           PASS                 0.65 
```
Fri May 29 17:31:11 2020 : Testscript running on redhat_8
Fri May 29 17:31:11 2020 : Successfully initialized testscript!
Fri May 29 17:31:11 2020 : VM Size: 
Fri May 29 17:31:19 2020 : Number of CPUs detected on VM: 8
Fri May 29 17:31:20 2020 : Generation: 1
Fri May 29 17:31:20 2020 : INFO: Guest services not found in lsvmbus information.
Fri May 29 17:31:20 2020 : Info: All VMBus device IDs have been found.
Fri May 29 17:31:20 2020 : Info: All VMBus device IDs have been found.
Fri May 29 17:31:20 2020 : Network driver is spread on all 8 cores as expected 8 cores.
Fri May 29 17:31:20 2020 : Network driver is spread on all 8 cores as expected 8 cores.
Fri May 29 17:31:20 2020 : Storage driver is spread on all 2 cores as expected 2 cores.
Fri May 29 17:31:20 2020 : Storage driver is spread on all 2 cores as expected 2 cores.
```

Reply:
Hi @LiliDeng ,
Do you think I should add a platform condition: "if Hyper-V, set VM_size=None."? I think it can prevent the failures if override size to Lv2 series in Hyper-V. Thanks!

Reply:
Hi @LiliDeng ,
Do you think I should add a platform condition: "if Hyper-V, set VM_size=None."? I think it can prevent the failures if override size to Lv2 series in Hyper-V. Thanks!